scripts/support.py

```diff
@@ -0,0 +1,29 @@
+try:
+    import __builtin__ as builtins
+except ImportError:
+    # Python 3...
+    import builtins
+
+"""
+This module exists to override the open function and log
+what files get opened.
+"""
+
+opened_files = []
+
+
+def clear_buffer():
+    global opened_files
+    opened_files = []
+
+
+oldopen = builtins.open
```

Comment:
I looked into different alternatives and couldn't find any better way to achieve this. This module is only loaded during unit testing, so it won't affect the build process outside of a testing context.